README.md

```diff
@@ -2,29 +2,74 @@
 
 A bookmarklet for monitoring GPT ad units and their associated targeting on web pages.
 
+* [Dev Requirements](#dev-requirements)
 * [Installation](#installation)
+* [Development](#development)
+* [Charles Setup](#charles-setup)
 * [Additional Information](#additional-information)
 * [Contributors](#contributors)
 
+## Dev Requirements
+- [Node](https://nodejs.org/en/)
+- [Charles](https://www.charlesproxy.com/)
+
 ## Installation
+`npm install`
 
-Create a bookmark in your browser. Paste the following into the "URL" field:
+## Development
+This project runs a simple http server that watches for changes to your JS/CSS files.
 
-```
-javascript:(function() {
-  var script = document.createElement('script');
-  script.src = 'https://forbes.github.io/ad-inspector/src/loader.js';
-  script.id = 'ad-inspector';
-  document.body.appendChild(script);
-})();
-```
+To start the server:
+`npm run start`
 
-## Additional Information
+Assuming you have no errors, you should be able to see your files on your local host (http://127.0.0.1:8080/).
+
+See proxied changes at https://forbes.github.io/ad-inspector/src/loader.js.
+
+## Charles Setup
+This used to be a straight-forward process, but our migration to HTTPS has complicated things.
```

Comment:
Let's remove this line -- this is some Forbes specific verbiage that no longer needs to be here.

Reply:
gotcha